adafruit_bme280/advanced.py

```diff
@@ -141,14 +142,14 @@ class Adafruit_BME280_Advanced(Adafruit_BME280):
     """
 
     # pylint: disable=too-many-instance-attributes
-    def __init__(self) -> None:
+    def __init__(self, proxy: typing.Union[I2C_Impl, SPI_Impl]) -> None:
```

Comment:
> Is this a proxy or perhaps a delegate? The data attribute in Adafruit_BME280 should be named accordingly.

If this question is about the name of the argument here. I do think maybe proxy isn't the best name. The thing that comes to mind for me is something like `bus_impl` or the full `bus_implementation`

Reply:
That sounds good. I renamed it to `bus_implementation`.